bug: fix policy structure and remove policyType

aschabus opened the pull request.

GitHubUser1339 reviewed (changes requested):
Please add changes to policy rules file as well.

aschabus commented:
@GitHubUser1339 The policy rules file is unchanged - I just changed the structure of the policy file and fixed a minor issue in the parameter file (using Out-FormattedPolicyDefinition.ps1)
